web/src/InstallationFinished.jsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import {
+  Button,
+  Title,
+  Text,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions
+} from "@patternfly/react-core";
+
+import Layout from "./Layout";
+import Center from "./Center";
+import { useInstallerClient } from "./context/installer";
+
+import {
+  EOS_TASK_ALT as InstallationFinishedIcon,
+  EOS_CHECK_CIRCLE as SectionIcon
+} from "eos-icons-react";
+
+const Actions = ({ onRestart, onFinish }) => (
+  <>
+    <Button isLarge variant="secondary" onClick={onRestart}>
+      Restart Installation
+    </Button>
+    <Button isLarge variant="primary" onClick={onFinish}>
+      Finish
+    </Button>
+  </>
+);
+
+function InstallationFinished() {
+  const client = useInstallerClient();
+  const onRestartAction = () => client.manager.startProbing();
+  const onFinishAction = () => console.log("FIXME: trigger a reboot machine action here!");
```

Comment:
probably should be manager action on dbus?

Reply:
Sure, but unless I'm wrong it is not available yet. That's why I put an example there. We can delete if by now if you want.